packages/client/src/models/ApisEndpoints.ts

```diff
@@ -46,7 +48,7 @@ export const DevApisEndpoints = (): ApisEndpoints => {
   }
 
   return {
-    Wroom: "http://wroom.test/",
+    Wroom: addTrailingSlash(wroomApi),
```

Comment:
Could you link the corresponding version or PR of the dev-env that will add this value please 🙏.
Otherwise it means we deploy a versions of SM that will no longer work with the dev-env.

Reply:
https://github.com/prismicio/prismicdev-cli/releases/tag/v0.4.2